meetfranz/franz
# fix(Recipes): Fix recipe download when adding a new service

When adding a new service, sometimes the tar.gz extraction failed. 
Fixes #67

### Description
- Replaced the, now deprecated, package `tar.gz` with `tar`.
- Improved recipe bulk download when multiple instances of a service, e.g. Slack are used. This had caused issues as well

### How Has This Been Tested?
1. Close Franz
2. Empty recipe directory
  - Mac: `~Library/Application Support/Franz/recipes/`
  - Windows: `%appdata%/Franz/recipes/`
3. Open Franz
4. Check if all recipes have been installed
5. Check if `recipes/temp` folder is empty

### Types of changes


### Checklist:


- [x] My code follows the code style of this project (run `$ yarn lint`).
